src/plugins/dashboard/public/dashboard_api/panels_manager.ts

```diff
@@ -284,7 +277,7 @@ export function initializePanelsManager(
       canRemovePanels: () => trackPanel.expandedPanelId.value === undefined,
       children$,
       duplicatePanel: async (idToDuplicate: string) => {
-        const panelToClone = await getDashboardPanelFromId(idToDuplicate);
+        const panelToClone = getDashboardPanelFromId(idToDuplicate);
 
         const duplicatedPanelState = await duplicateReactEmbeddableInput(
```

Comment:
Can remove `await` when calling `duplicateReactEmbeddableInput`

Reply:
Removed in [567ffc3](https://github.com/elastic/kibana/pull/203662/commits/567ffc39a88f600130541b55e8a2da15c457c27c)